src/Nest/CommonOptions/Stats/TranslogStats.cs

```diff
@@ -8,11 +8,23 @@ public class TranslogStats
 		[JsonProperty("operations")]
 		public long Operations  { get; set; }
 
+		[JsonProperty("uncommitted_operations")]
+		public long UncommittedOperations  { get; set; }
```

Comment:
slight nit: Should this be an `int`?

https://github.com/elastic/elasticsearch/blob/6.3/server/src/main/java/org/elasticsearch/index/translog/TranslogStats.java#L37